### ISSUE TITLE
Triggering switches enables feature with a unwanted timer

### ISSUE THREAD

**Issue by Rolrik** [mathieu-mp/intex-spa#16]
Triggering `async_set_filter` enables filter with a 2 hours timer, instead of enabling filter for good.

**Comment by mathieu-mp** [comment]
Hi @Rolrik ,

I read the intex mobile app java code again, and there seem to be no way to actuate the switch without triggering the associated timer.
Thus, I am sorry but I just cannot change the way the python package works.

If anyone has a suggestion about a way to solve this issue, don't hesitate to comment and reopen.

**Comment by Mira43** [comment]
Hello @mathieu-mp i don't know if you are on the project at this date, but i discovered something that i never know on github or other websites that talk to the intex spa protocol.
But i used the schedule part of the mobile app. When turning on/off or modifiy the schedule rule, i intercepted the message, tried to resend over telnet chanel after disabled the rule. The Spa started at the good time to heat with the good parameters i set in the rule.

This to share with you an example, and maybe understand how it works better, and perhaps, suggest to add schedule features in your library ?

So, basically, this is my heater rule i set in the mobile app : 
![image](https://github.com/user-attachments/assets/2db177ea-a72c-4164-8a91-3f124b8e3a17)
So i set every day at 4:10 PM, heat until 38°C for 8 hours.

I intercepted this command message, and replay it later, it worked : 
{"data":"8888280F0A55000808FF2600000000000000000000000000000000000000000000000000000000000022","sid":"1724248867138","type":1}

Hope it can help you or another person to understand what is the logic with the schedules ...
